client/src/containers/explore-map/component.tsx

```diff
@@ -24,6 +24,16 @@ const ExploreMap = () => {
       <Media greaterThanOrEqual="sm">
         <Sidebar />
       </Media>
+      <Media greaterThanOrEqual="sm">
+        <a
+          href="https://www.google.es/"
+          target="_blank"
+          rel="noopener noreferrer"
+          className="absolute bottom-1/2 -right-9 z-10 -rotate-90 rounded-md border border-navy-500 bg-white px-2 pb-4 text-sm hover:-translate-x-2 hover:transition-transform"
```

Comment:
if you put `hover:transition-transform` only on hover, the animation will only happen on the enter animation. The exit animation is not animating right now